test/e2e/functional/tests/e2e.go

```diff
@@ -428,7 +421,13 @@ var _ = Describe("Ingress Node Firewall", func() {
 			infwutils.DefineWithInterface(inf, "eth0")
 			infwutils.DefineDenyTCPRule(inf, "1.1.1.1/32", 40000)
 			Expect(testclient.Client.Create(context.Background(), inf)).To(Succeed())
-			Expect(testclient.Client.Delete(context.Background(), inf)).To(Succeed())
+			Eventually(func() bool {
```

Comment:
move this green block into a small helper functoin and then call this as a one liner to avoid code duplication?